src/client/Microsoft.Identity.Client.Broker/RuntimeBroker.cs

```diff
@@ -106,20 +111,18 @@ private void LogEventRaised(NativeInterop.Core sender, LogEventArgs args)
             LogLevel msalLogLevel = LogLevelMap[args.LogLevel];
             if (_logger.IsLoggingEnabled(msalLogLevel))
             {
-                string msgWithPii = string.Empty;
-                string msgNoPii = string.Empty;
+                string msgWithPii = _logger.PiiLoggingEnabled ? args.Message : string.Empty;
+                string msgNoPii = !_logger.PiiLoggingEnabled ? args.Message : string.Empty;
 
-                // as of now, there will be no Pii.
-                // TODO change when Pii is added
-                // https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3822
-                _logger.Log(msalLogLevel, string.Empty, args.Message);
+                _logger.Log(msalLogLevel, msgWithPii, msgNoPii);
```

Comment:
nit: maybe something like this instead? This way we don't allocate a duplicate string each time; which may be an issue if LogEventRaised gets called a lot.

```csharp
                if (_logger.PiiLoggingEnabled)
                {
                    _logger.Log(msalLogLevel, args.Message, string.Empty);
                }
                else
                {
                    _logger.Log(msalLogLevel, string.Empty, args.Message);
                }
```

Reply:
Done.